### ISSUE TITLE
Can't get song browser mod to work but all other mods do work

### ISSUE THREAD

**Issue by Trist04160** [halsafar/BeatSaberSongBrowser#170]
I've been installing all the updated mods that I use that I've been finding and song browser is the only mod that doesn't want to work. I tried opening IPA.exe but that didn't do anything and I've tried a load of other stuff but nothing has worked. I don't think any of my other mods are keeping it from working so my last option seems to just be to delete all my mods and try song browser by itself to see if it works and then add the rest of my mods. I hope this is a bug or an issue and it's not just me because I don't really know the exact process of adding updated mods when beat saber updates. This is my first time having to do this so I'm not 100% sure that I did the process correctly. I hope someone can help.

**Comment by Artiswastaken** [comment]
I'm having the same issue, even with customjsondata installed the mod just doesn't show up ingame

**Comment by putnamto** [comment]
in the same boat as you guys, this is my second favorite mod and i was excited to see its updated, but it doesnt work :(

**Comment by Trist04160** [comment]
Okay, I FINALLY solved it, at least for me. So I did two things at once so I don't know which fixed it but I did both anyways. So first I went and downloaded Song Browser again and this time just took the .dll file out of the zip and moved it to the plugins folder instead of extracting the zip. And then the other thing I did was look for another mod called Playlist manager which I needed back and downloaded it and extracted that to my beat saber folder. I then opened the game and I saw the song browser mod working!! I feel like it was the playlist manager that fixed it for me at least, but I don't care which fixed it I'm just happy it's working again! I really hope this also helps you guys and others with the same problem!
